pkg/compactor/retention/retention_test.go

```diff
@@ -675,6 +684,9 @@ func TestMarkForDelete_SeriesCleanup(t *testing.T) {
 			expectedModified: []bool{
 				false,
 			},
+			numChunksDeleted: []int64{
+				0,
```

Comment:
by the `name` of the test case here shouldn't it be 1?

Reply:
Thanks for taking a look! This tc is testing a case where there is a delete request with a line filter but the filter is not matching any lines. Only the lines which are matched by filter would get deleted so essentially it is a no-op.